src/components/text-editor/text-editor.tsx

```diff
@@ -153,6 +154,14 @@ export class TextEditor implements FormComponent<string> {
     @Prop({ reflect: true })
     public ui?: 'standard' | 'minimal' = 'standard';
 
+    /**
+     * Set to `true` to allow parsing of table data. Only works when `type` is `html`.
+     * @private
+     * @alpha
+     */
+    @Prop({ reflect: true })
+    public enableTables?: boolean;
```

Comment:
Why is this prop named differently than the one in the prosemirror adaptor?

Reply:
A consequence of tiredness methinks that I didn't notice 👀 a fixup is incoming.

Reply:
> Also, could we enable it per default if the content type is html? I'm not sure we should treat the interface here as some kind of feature switch. If we don't of course want to treat this as something that different consumers would like to turn on or off because of some good reasons

I considered that. My thinking for not removing the extra prop was in relation to tables not being supported in Markdown at all and us providing the possibility to swap between `html` and `markdown`. Having an explicit `enableTables` felt like it would be good to know that the customer did indeed specifically want tables enabled. 

I suppose it might be un-realistic to expect consumers to swap between `html` and `markdown` with any regularity. I'm happy to remove the `enableTables` prop and simply enable it if `html` is the default `contentType` that is passed in.

Reply:
I'd vote for enabling the plugin if the output type is `html`.

What happens if the plugin is loaded and the output is changed to `markdown`?

(At the face of it, saying that the output type has to be set from the outset, and not supporting later changes, doesn't seem completely absurd to me. As long as we document it.)

Reply:
See [52ce333](https://github.com/Lundalogik/lime-elements/pull/3301/commits/52ce3335351e7850e0845bc0e15fc23299d888e8)